sdk/bootstrap/main.go

```diff
@@ -336,52 +336,43 @@ func isFile(path string) (bool, error) {
 	return false, nil
 }
 
-func GetLocalIPold() string {
-	addrs, err := net.InterfaceAddrs()
-	if err != nil {
-		return ""
-	}
-	for _, address := range addrs {
-		// check the address type and if it is not a loopback the display it
-		if ipnet, ok := address.(*net.IPNet); ok && !ipnet.IP.IsLoopback() {
-			if ipnet.IP.To4() != nil {
-				return ipnet.IP.String()
-			}
-		}
-	}
-	return ""
-}
-
-func GetLocalIP() (addr string, err error) {
+func getContainerIPAddress() (string, error) {
 	ip, err := ContainerIP()
-
 	if err != nil {
-		return ip.String(), err
+		return "", err
 	}
 
-	addr = ip.String()
-	return ip.String(), err
+	return ip.String(), nil
 }
 
 // main
 
 func main() {
 	args := parseArgs()
 
-	pod_ip, err := GetLocalIP()
+	log.Printf("Looking up container IP address...")
+	containerIP, err := getContainerIPAddress()
 	if err != nil {
-		log.Fatalf("Cannot find the container's IP address: ", err)
+		log.Fatalf("Cannot look up container's IP address: %s", err)
+	}
+	log.Printf("Container IP address was found to be: %s", containerIP)
+	// Set both LIBPROCESS_IP and MESOS_CONTAINER_IP so that SDK developers
```

Comment:
Thise could also be done in a loop like @asridharan suggested in #1515.

Reply:
I personally don't like the loop style, as someone can easily swap the order of an array.

Reply:
(it also IMO sacrifices readability for saving a handful of lines)